modules/dr-restoring-cluster-state.adoc

```diff
@@ -281,26 +281,29 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
-. If you are using the `OVNKubernetes` network plugin, delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
+. If you are using the `OVNKubernetes` network plugin, you must restart `ovnkube-controlplane` pods.
+.. Delete all the `ovnkube-controlplane` pods by running the following command:
 +
 [source,terminal]
 ----
-$ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
+$ oc -n openshift-ovn-kubernetes delete pod -l app=ovnkube-control-plane
 ----
-
-. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed to restart the Open Virtual Network (OVN) Kubernetes pods on all of the hosts in the next step.
+.. Verify all of the `ovnkube-controlplane` pods were redeployed by running the following command:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes get ds/ovnkube-master -o yaml | grep -E '<non-recovery_controller_ip_1>|<non-recovery_controller_ip_2>'
+$ oc -n openshift-ovn-kubernetes get pod -l app=ovnkube-control-plane
 ----
+
+. If you are using the `OVNKubernetes` network plugin, restart the Open Virtual Network (OVN) Kubernetes pods on all the nodes one by one. Use the following steps to restart OVN-Kubernetes pods on each node:
 +
-[NOTE]
+[IMPORTANT]
 ====
-It can take at least 5-10 minutes for the OVN-Kubernetes control plane to be redeployed and the previous command to return empty output.
```

Comment:
@palonsoro does this need to be added back in as well?

Reply:
I don't think so. The ovn-kubernetes pods are now much more lightweight. This sentence is present in 4.13 and older because CNO has to re-deploy the control plane single replica and it intentionally waits several minutes before doing so. That's no longer the case.